lochness/transfer/__init__.py

```diff
@@ -479,6 +479,34 @@ def lochness_to_lochness_transfer_s3_protected(Lochness):
                 logger.debug(command_str)
                 logger.debug('aws rsync completed')
 
+    # interview run sheets
+    # phoenix_root / PROTECTED / site / raw / subject / datatype
+    interview_dirs = Path(Lochness['phoenix_root']).glob(
+        f'PROTECTED/*/raw/*/interviews')
+
+    for interview_dir in interview_dirs:
+        s3_target = re.sub(Lochness['phoenix_root'],
+                           s3_phoenix_root,
+                           str(interview_dir))
+        command = f"aws s3 sync \
+                {interview_dir} \
+                s3://{s3_bucket_name}/{s3_target} --delete \
+                --exclude='*' --include='*Run_sheet_interviews_*.csv'"
```

Comment:
Only the a/v run sheets would be transferred to staging. It's missing in staging because RPMS export only recently included run sheets for interviews. More updates to come for this in next PR.